egs/wsj/s5/steps/nnet3/make_bottleneck_features.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+#           2016  Pegah Ghahremani
+# Apache 2.0
+# This script dumps bottleneck feature for model trained using nnet3.
+
+# Begin configuration section.
+stage=1
+nj=4
+cmd=run.pl
+use_gpu=false
+bnf_name=Tdnn_Bottleneck_renorm
```

Comment:
As I pointed out in my previous review, it is better not to use `bnf_name` as an optional argument. Please consider making it compulsory.